error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ClassRulesTestInput.java

```diff
@@ -23,4 +24,13 @@ Predicate<String> testClassReferenceIsInstancePredicate() throws IOException {
     Class<?> clazz = CharSequence.class;
     return s -> clazz.isInstance(s);
   }
+
+  Function<Number, Integer> testClassLiteralCast() {
+    return i -> (Integer) i;
+  }
+
+  Function<Number, Integer> testClassReferenceCast() {
+    Class<? extends Integer> clazz = Integer.class;
+    return i -> clazz.cast(i);
```

Comment:
We don't need the separate variable here. (It's used above to test for a _non_-matching scenario.)

Reply:
Ah! Makes sense 👍